src/Watchy.cpp

```diff
@@ -579,6 +582,17 @@ void Watchy::drawWatchFace() {
   display.println(currentTime.Minute);
 }
 
+void Watchy::button1()
+{
+}
+void Watchy::button2()
+{
+}
+void Watchy::button3()
+{
+}
+
+
```

Comment:
This will help keep the style consistent:

```suggestion
void Watchy::handleButton2Press() {}
void Watchy::handleButton3Press() {}
void Watchy::handleButton4Press() {}

```